Makefile

```diff
@@ -143,44 +117,42 @@ ifeq ("$(wildcard ./toolchains/$(CROSS_ARCH).cmake)","")
 	$(error invalid cross-compile target '$(CROSS_ARCH)')
 endif
 
-.PHONY: build/$(CROSS_ARCH)/sdk/bin/toit.run build/$(CROSS_ARCH)/sdk/bin/toit.compile
-build/$(CROSS_ARCH)/sdk/bin/toit.run build/$(CROSS_ARCH)/sdk/bin/toit.compile: build/$(CROSS_ARCH)/CMakeCache.txt
-	(cd build/$(CROSS_ARCH) && ninja build_tools)
-
-build/$(CROSS_ARCH)/CMakeCache.txt: build/$(CROSS_ARCH)/
-	(cd build/$(CROSS_ARCH) && cmake ../../ -G Ninja -DCMAKE_BUILD_TYPE=Release -DCMAKE_TOOLCHAIN_FILE=../../toolchains/$(CROSS_ARCH).cmake --no-warn-unused-cli)
+build/$(CROSS_ARCH)/CMakeCache.txt:
+	mkdir -p build/$(CROSS_ARCH)
+	(cd build/$(CROSS_ARCH) && cmake ../../ -G Ninja -DTOITC=$(CURDIR)/build/host/sdk/bin/toit.compile -DCMAKE_BUILD_TYPE=Release -DCMAKE_TOOLCHAIN_FILE=../../toolchains/$(CROSS_ARCH).cmake --no-warn-unused-cli)
```

Comment:
Moved, but also added the `-DTOITC=$(CURDIR)/build/host/sdk/bin/toit.compile`.

This way the cross architecture uses the local host to compile snapshots.

Reply:
Nice cleanup; much easier to maintain.